sections/attributes.include

```diff
@@ -542,7 +542,7 @@
     </tr>
     <tr>
       <th><code>nonce</code></th>
-      <td><{script}>; <{style}></td>
+      <td><{link}>; <{script}>; <{style}></td>
```

Comment:
There are two indices in the WHATWG spec: one for attribute -> element, and one for the inverse. I don't know if the W3C doc follows that model.